apps/portal-front/src/components/service/components/header/service-list-header.tsx

```diff
@@ -8,7 +8,7 @@ import React from 'react';
 
 export enum ServiceListFilterKey {
   Label = 'label',
-  IntegrationsType = 'integration_feed_type',
```

Comment:
There is still one translation with this name `(integration_feed_type)` to refactor 😉